ui/src/app/app.component.ts

```diff
@@ -206,16 +205,13 @@ export class AppComponent {
           case ConnectionFailure.Network:
             message = 'Phone or computer has no network connection.'
             break
-          case ConnectionFailure.Diagnosing:
-            message = new IonicSafeString('Running network diagnostics <ion-spinner style="padding: 0; margin: 0" name="dots"></ion-spinner>')
-            break
           case ConnectionFailure.Tor:
             message = 'Browser unable to connect over Tor.'
-            link = 'https://docs.start9.com/support/FAQ/setup-faq.html#tor-failure'
+            link = 'https://docs.start9.com/support/FAQ/troubleshooting.html#tor-failure'
```

Comment:
Currently a 404, are we gonna fix this
![image](https://user-images.githubusercontent.com/2364004/146849220-b88da45d-180e-44d7-81b8-86211108b6aa.png)

Reply:
Yes this is expected at present. All UI links will be fixed before deployment